libraries/AP_NavEKF2/AP_NavEKF2_core.cpp

```diff
@@ -96,6 +97,52 @@ bool NavEKF2_core::setup_core(NavEKF2 *_frontend, uint8_t _imu_index, uint8_t _c
 
     return true;
 }
+
+void NavEKF2_core::switchGyros(uint8_t _new_gyro_index)
+{
+    if (gyro_index != _new_gyro_index) {
+        // NOTE: theoretically, we should wait until we reach the end of the IMU buffer prior to resetting these
+        stateStruct.gyro_bias.zero();
```

Comment:
I'm concerned about a very short IMU outage (1 ms timeout for example) causing biases to reset